### PR TITLE
Add esbuild-plugin-pnp

### PR DESCRIPTION
**Pull request by eigilsagafos**

### CODE REVIEW

**Comment by AlexxNB** [comment]
You forgot rename link title =)

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This is just a centralized list of 3rd-party plugins to make discovery easier. N
 * [esbuild-graphql-loader](https://github.com/luckycatfactory/esbuild-graphql-loader): A plugin allowing for GraphQL file imports.
 * [esbuild-plugin-cache](https://github.com/dalcib/esbuild-plugin-cache): A plugin to cache http/https modules. It works with [import-maps](https://github.com/WICG/import-maps)
 * [esbuild-plugin-flow](https://github.com/dalcib/esbuild-plugin-flow): A plugin to strip types for Flow files using flow-remove-types package.
+* [esbuild-plugin-flow](https://github.com/shiftx/esbuild-plugin-pnp): A plugin to support yarn 2 (berry) with PnP (Plug’n’Play).
```
Comment (evanw) [inline README.md:12]: This should say `esbuild-plugin-pnp` not `esbuild-plugin-flow`.

**Review by evanw — COMMENTED** [review]
Thanks for creating this plugin! Just one thing to fix...

File: README.md (hunk 1 above)
Comment (eigilsagafos) [reply README.md]: 👍 Thanks @evanw

**Pull request merged**